config/internal/common/default/mariadb-access-policy.yaml.tmpl

```diff
@@ -0,0 +1,23 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: mariadb-{{.Name}}
+  namespace: {{.Namespace}} 
+spec:
+  podSelector:
+    matchLabels:
+      app: mariadb-{{.Name}}
```

Comment:
There seems to be a trailing whitespace in your file, which is why the pre-commit is failing. Tip: you could use the following command to detect trailing whitespaces:

`grep -rn '[[:blank:]]$'  <your-file-path`

or just set up your IDE to find whitespaces.